### DISCUSSION TITLE
Enable automated pre-translation via MT on Crowdin

### DISCUSSION THREAD

**Discussion by n-stein** [moneymanagerex/moneymanagerex#6591] [Ideas]
There are many incomplete languages on Crowdin, and every time we add/change a translated string it creates more work for translators. 

Crowdin enables the use of machine translators to pre-translate any strings which are not currently translated. Then, users can edit/approve the strings if they feel they have a more accurate translation. It will also auto-translate any newly added strings so that UI changes are localized instantly.

Perhaps we should enable this. Thoughts?

**Comment by DevAnyKR** [comment]
MMEX help page has disappeared from crowdin. It's not translable.
has the Korean help I translated been deleted forever?

**Reply by vomikan** [reply]
https://crowdin.com/project/moneymanagerex/ko

**Reply by n-stein** [reply]
> MMEX help page has disappeared from crowdin. It's not translable.

@vomikan please correct me if I'm wrong, but as far as I am aware the help pages were never translated via Crowdin. They must be translated in the HTML directly in the docs folders https://github.com/moneymanagerex/moneymanagerex/tree/master/docs

To add a language you would copy the en_GB folder to a new ko_KR folder, then translate the text in all of the .html files and replace the screenshots if necessary.

**Reply by DevAnyKR** [reply]
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/d7019455-15a2-40dd-9133-27393032b7ec)

I completed the Korean translation of MMEX help page in CROWDIN site last year. (2023-04~2023-05)

**Reply by vomikan** [reply]
At one time, help pages were available on crowdin. But, since this is a very large volume, they were practically not translated. We decided to remove them.

**Comment by DevAnyKR** [comment]
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/4e11183c-c655-4ca8-a875-f593b3042400)

Even if automatic translation is applied, the button letters above will not be translated, since they are not registered in CROWDIN...
